docs/docs/tutorials/providers.md

```diff
@@ -0,0 +1,613 @@
+
+    ```js
+    const { Web3 } = require('web3');
+
+    // Connect to the Ethereum network using the HTTP provider
+    const ganacheUrl = 'http://localhost:7545';
+    const httpProvider = new Web3.providers.HttpProvider(ganacheUrl);
+    const web3 = new Web3(httpProvider);
+
+    async function main() {
+    	try {
+    		// Get the current block number from the network
+    		const currentBlockNumber = await web3.eth.getBlockNumber();
+    		console.log('Current block number:', currentBlockNumber);
+
+    		// Get the list of accounts in the connected node (e.g., Ganache)
+    		const accounts = await web3.eth.getAccounts();
+
+    		// Send a transaction to the network and wait for the transaction to be mined.
+    		// Note that sending a transaction with Ganache will cause it, in its default configuration, to min a new block.
+    		const transactionReceipt = await web3.eth.sendTransaction({
+    			from: accounts[0],
+    			to: accounts[1],
+    			value: web3.utils.toWei('0.001', 'ether'),
+    		});
+    		console.log('Transaction Receipt:', transactionReceipt);
+
+    		// Get the updated block number
+    		const updatedBlockNumber = await web3.eth.getBlockNumber();
+    		console.log('Updated block number:', updatedBlockNumber);
+    	} catch (error) {
+    		console.error('An error occurred:', error);
+    	}
+    }
+
+    main();
+    ```
+
+6. Ensure that Ganache is running as mentioned in the [Prerequisites](#prerequisites) section.
+
+7. In the command prompt or terminal window, type `node web3-http-provider.js` and press Enter. This will run your JavaScript file and connect to the Ethereum network using the HTTP provider and Ganache.
+
+If everything is set up properly, you should see the current block number, the transaction receipt, and the updated block number printed in the console:
+
+```bash
+Current block number: 0n
+Transaction Receipt: {
+  transactionHash: '0x0578672e97d072b4b91773c8bfc710e4f777616398b82b276323408e59d11362',
+  transactionIndex: 0n,
+  blockNumber: 1n,
+  blockHash: '0x348a6706e7cce6547fae2c06b3e8eff1f58e4669aff88f0af7ca250ffdcdeef5',
+  from: '0x6e599da0bff7a6598ac1224e4985430bf16458a4',
+  to: '0x6f1df96865d09d21e8f3f9a7fba3b17a11c7c53c',
+  cumulativeGasUsed: 21000n,
+  gasUsed: 21000n,
+  logs: [],
+  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+  status: 1n,
+  effectiveGasPrice: 2000000000n,
+  type: 0n
+}
+Updated block number: 1n
+```
+
+### WebSocket Provider
```

Comment:
```suggestion
```

This example showed how to connect to the Ethereum network using HTTP provider, get the current block number, send a transaction, and get the updated block number.

### WebSocket Provider
```